### PR TITLE
Add `--no-cache` docs

### PR DESCRIPTION
**Pull request by dchw**

### CODE REVIEW

File: docs/earthfile/earthfile.md [hunk 1]
```diff
@@ -139,6 +139,16 @@ Push commands were introduced to allow the user to define commands that have an
 
 Note that non-push commands are not allowed to follow a push command within a recipe.
 
+#### `--no-cache`
+
+Marks the command as one that will run every time; ignoring any cache. Any commands following the first invocation of a `RUN` with `--no-cache` will also not use the cache, so it is iporatnt to use this command where it will not cause a lot of duplicate work.
```
Comment (alexcb) [inline docs/earthfile/earthfile.md:144]:
```suggestion
Force the command to run every time; ignoring any cache. Any commands following `RUN` commands regardless of the `--no-cache` will also ignore the cache.
```

File: docs/earthfile/earthfile.md [hunk 2]
```diff
@@ -139,6 +139,16 @@ Push commands were introduced to allow the user to define commands that have an
 
 Note that non-push commands are not allowed to follow a push command within a recipe.
 
+#### `--no-cache`
+
+Force the command to run every time; ignoring any cache. Any commands following `RUN` commands regardless of the `--no-cache` will also ignore the cache.
```
[the remark this answers is not in the document]
Comment (alexcb) [reply docs/earthfile/earthfile.md:144]: maybe the wording still needs some slight tweaking. :shrug:

Comment (dchw) [inline docs/earthfile/earthfile.md:144]:
```suggestion
Force the command to run every time; ignoring any cache. Any commands following the invocation of `RUN --no-cache`, will also ignore the cache.
```

[the remark this answers is not in the document]
Comment (dchw) [reply docs/earthfile/earthfile.md:144]: I just made a similar suggestion. Willing to go with whatever you like.

[the remark this answers is not in the document]
Comment (dchw) [reply docs/earthfile/earthfile.md:144]: Its not _just_ `RUN` that ignores cache afterward; its every command.

[the remark this answers is not in the document]
Comment (alexcb) [reply docs/earthfile/earthfile.md:144]: ah right, sounds good then.

File: docs/earthfile/earthfile.md [hunk 3]
```diff
@@ -139,6 +139,16 @@ Push commands were introduced to allow the user to define commands that have an
 
 Note that non-push commands are not allowed to follow a push command within a recipe.
 
+#### `--no-cache`
+
+Force the command to run every time; ignoring any cache. Any commands following the invocation of `RUN --no-cache`, will also ignore the cache.
+
+If a target that uses `--no-cache` is `COPY`-ed from; all artifacts saved after the `--no-cache` will not be use the cache when copying.
```
Comment (vladaionescu) [inline docs/earthfile/earthfile.md:146]:
```suggestion
If a target that uses `--no-cache` is `COPY`-ed from; all artifacts saved after the `--no-cache` will not use the cache when copying.
```

File: docs/earthfile/earthfile.md [hunk 4]
```diff
@@ -139,6 +139,16 @@ Push commands were introduced to allow the user to define commands that have an
 
 Note that non-push commands are not allowed to follow a push command within a recipe.
 
+#### `--no-cache`
+
+Force the command to run every time; ignoring any cache. Any commands following the invocation of `RUN --no-cache`, will also ignore the cache.
```
Comment (vladaionescu) [inline docs/earthfile/earthfile.md:144]: I'm thinking if it's better to only keep this paragraph.

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**